src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
```

Comment:
It doesn't seem to be any benefit into defining that category ("dynamic" commands). You delegate immediately to the external command when you call the very thin wrapper function `dynamic_cli_command`. 

This CLI here suffices to be aware of two categories of commands. COS and Enterprise (there's another PR in progress to add control center as an example of an enterprise-only command).